src/frontends/common/include/openvino/frontend/extension/op.hpp

```diff
@@ -145,80 +186,150 @@ class OpConversionFunction {
     std::map<std::string, ov::Any> m_attr_values_map;
 };
 
+class OpConversionFunctionNamed {
+public:
+    explicit OpConversionFunctionNamed(const std::function<std::shared_ptr<ov::Node>()>& op_creator,
+                                       const std::vector<std::string>& in_names_vec,
+                                       const std::vector<std::string>& out_names_vec,
+                                       const std::map<std::string, std::string>& attr_names_map = {},
+                                       const std::map<std::string, ov::Any>& attr_values_map = {})
+        : m_op_creator(op_creator),
+          m_in_names_vec(in_names_vec),
+          m_out_names_vec(out_names_vec),
+          m_attr_names_map(attr_names_map),
+          m_attr_values_map(attr_values_map) {}
+
+    std::map<std::string, OutputVector> operator()(const NodeContext& context) {
+        auto node = m_op_creator();
+
+        std::vector<Output<Node>> inputs;
+        for (const auto& name : m_in_names_vec) {
+            for (size_t i = 0; i < context.get_input_size(name); ++i) {
+                inputs.push_back(context.get_input(name, i));
+            }
+        }
+
+        node->set_arguments(inputs);
+        FWVisitor fw_visitor(context, m_attr_names_map, m_attr_values_map);
+        node->visit_attributes(fw_visitor);
+        node->validate_and_infer_types();
+        std::map<std::string, OutputVector> out;
+        OPENVINO_ASSERT(m_out_names_vec.size() == node->get_output_size(),
+                        "each output should has a name, names number: ",
+                        m_out_names_vec.size(),
+                        ", output size: ",
+                        node->get_output_size());
+        int i = 0;
+        for (const auto& name : m_out_names_vec) {
+            if (out.find(name) == out.end()) {
+                out.insert({name, OutputVector()});
+            }
+            out[name].emplace_back(node->output(i++));
+        }
+        return out;
+    }
+
+private:
+    std::function<std::shared_ptr<ov::Node>()> m_op_creator;
+    std::vector<std::string> m_in_names_vec;
+    std::vector<std::string> m_out_names_vec;
+    std::map<std::string, std::string> m_attr_names_map;
+    std::map<std::string, ov::Any> m_attr_values_map;
+};
+
+template <typename BaseConversionType>
+std::shared_ptr<ov::Node> OpExtensionBase<BaseConversionType, void>::make_node(const std::string& ov_type_name) {
```

Comment:
Why do we need this helper?

Reply:
The content of make_node comes from 'OpExtensionBase<BaseConversionType, void>' constructor. The 'make_node' will be called in the original OpExtensionBase constructor and the new added constructor for paddle.